experimenter/experimenter/experiments/constants.py

```diff
@@ -84,6 +84,23 @@ def get_kinto_collection_for(self, experiment: NimbusExperiment) -> str:
                 for feature in experiment.feature_configs.all()
             }
 
+            min_version = (
+                NimbusConstants.Version.parse(experiment.firefox_min_version)
+                if experiment.firefox_min_version != NimbusConstants.Version.NO_VERSION
+                else None
+            )
+
+            if min_version is not None and min_version >= NimbusConstants.Version.parse(
+                NimbusConstants.DESKTOP_KINTO_COLLECTION_RIDE_ALONG_MIN_VERISON
+            ):
+                # Publish to the alternate collection if there is only a single
+                # non-default collection in the set.
+                if (
+                    len(target_collections) == 2
+                    and self.default_kinto_collection in target_collections
+                ):
+                    target_collections.remove(self.default_kinto_collection)
```

Comment:
Couldn't we say this slightly differently like

```py
if len(target_collections) > 1 and secure_collection in target_collections:
    target_collections = {secure_collection}
```

so that if ANYTHING requires the secure collection then the whole thing gets lifted up into secure?